### PR TITLE
Features/users

### PR DESCRIPTION
**Pull request by MartinJAA**
Hi team, I'm adding crud keywords for features users and test cases

### CODE REVIEW

File: libraries/process_data/url_assembler.py
Comment (Rochius) [inline libraries/process_data/url_assembler.py:1]: Hi Martín! you had commented lines in this file, if are not necessary delete it please

File: wp_api/tests/users/users.py
Comment (Rochius) [inline wp_api/tests/users/users.py:1]: Hi Martín! is necessary this file?

File: wp_api/tests/users/tb52_users.robot
Comment (Rochius) [inline wp_api/tests/users/tb52_users.robot:1]: Hi Martín! Please add the blank line at the end of the file

Comment (Rochius) [reply wp_api/tests/users/tb52_users.robot:1]: Is necessary the variable ${end_point_path}?

File: libraries/process_data/url_assembler.py
Comment (Rochius) [reply libraries/process_data/url_assembler.py:1]: LGTM!

File: wp_api/tests/users/users.py
Comment (Rochius) [reply wp_api/tests/users/users.py:1]: LGTM!

File: wp_api/tests/users/tb52_users.robot
Comment (Rochius) [reply wp_api/tests/users/tb52_users.robot:1]: LGTM!

**Review by Rochius — APPROVED** [review]

File: libraries/process_data/url_assembler.py
Comment (MartinJAA) [reply libraries/process_data/url_assembler.py:1]: Hi Rocio! Thank you for your observation, I fixed it

File: wp_api/tests/users/users.py
Comment (MartinJAA) [reply wp_api/tests/users/users.py:1]: Hi Rocio! Thank you for your observation, I deleted it

File: wp_api/tests/users/tb52_users.robot
Comment (MartinJAA) [reply wp_api/tests/users/tb52_users.robot:1]: Hi Rocio! Thank you for your observation, I added the lines and deleted that unnecessary variable

**Review by Rogerrenjifo — APPROVED** [review]

**Review by Rogerrenjifo — APPROVED** [review]
LGTM. Good job my friend.

**Review by Rogerrenjifo — APPROVED** [review]
LGTM.

**Review by TelmaRF — APPROVED** [review]
LGTM! good job

**Pull request merged**